src/components/List/RowOptionsLink.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import classnames from "classnames";
+
+import { Column } from "../Grid";
+import { Link } from "../Text";
+import { mediumAndUp } from "../../theme/mediaQueries";
+import constants from "../../theme/constants";
+
+export const DisplayWhen = ({ predicate, children }) => {
```

Comment:
We want to move away from using `DisplayWhen` as it doesn't short circuit logic from executing within its `children`. It would be preferable to use boolean logic that short circuits instead:

E.g. 
```
{variant === "withLink" && 
    <RowOptionsLink
         variant={variant}
         isOpen={isOpen}
         url={linkUrl}
         index={index}
         onClick={onOverflowClick}
     >
         {linkTitle}
     </RowOptionsLink>
}
```

Reply:
Done.